bentoml/service/env.py

```diff
@@ -135,9 +131,15 @@ class BentoServiceEnv(object):
             pip dependencies and pin their version
         requirements_txt_file: path to the requirements.txt where pip dependencies
             are explicitly specified, with ideally pinned versions
-        conda_channels: list of extra conda channels to be used
-        conda_overwrite_channels: Turn on to make conda_channels overwrite the list of
-            channels instead of adding to it
+        conda_channels: list of extra conda channels other than dafault channels to be
+            used. This is equivalent to passing the --channels to conda commands.
+            If the `conda_env_yml_file` is specified, this will override the `channels`
```

Comment:
Not a change proposed in this PR, but I realized the current behavior of `conda_env_yml_file` might not be ideal, should BentoML just ignore the `conda_channels`, `conda_dependencies`  and `conda_override_channels` here if the user has provided a  `conda_env_yml_file`?